packages/vscode-extension/src/service/TranslationService.ts

```diff
@@ -0,0 +1,31 @@
+/*
+** Copyright (c) 2019 Oracle and/or its affiliates.  All rights reserved.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+const ApplicationConstants = require('../../../node-cli/src/ApplicationConstants');
+const FileUtils = require('../../../node-cli/src/utils/FileUtils');
+const path = require('path');
+let MESSAGES: { [x: string]: any; };
+
+export class TranslationService {
```

Comment:
why do we have this as a class? wouldn't it be better to have a utils file, something like:
```
const ApplicationConstants = require('../../../node-cli/src/ApplicationConstants');
const FileUtils = require('../../../node-cli/src/utils/FileUtils');
const path = require('path');

const filePath = path.join(__dirname, ApplicationConstants.DEFAULT_MESSAGES_FILE);
let MESSAGES: { [x: string]: any; } = FileUtils.readAsJson(filePath);

injectParameters( ...

export getMessage( ...
```

So that, outside this file we can avoid:
```
import { TranslationService } from '... TranslationService'

const translationService = new TranslationService()
translationService.getMessage(...
```
and change it for:
```
import { getMessage } from '... TranslationService'

getMessage(...
```

Reply:
I like your approach, I only replicate the behavior of TranslationService from Node.js CLI. Maybe it will be interesting to make something similar there?

Reply:
We will reuse TranslationService as in  node-cli package  as discussed in the call. I close this conversation as we have already other opened.